kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: osc-playground
+
+resources:
+  - ../../../../base/jupyterhub
+
+patchesJson6902:
+  - patch: |
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/overlays/-
+        value: storage-class
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/parameters/-
+        value:
+          name: storage_class
+          value: ocs-storagecluster-cephfs
```

Comment:
Just noticed.. maybe you wanna use NFS?

Reply:
yes I do good catch

Reply:
I copied it from stage...which should also probably be nfs

Reply:
updated

Reply:
we can just remove it and let it use the default storageclass (which currently is the nfs)